lib/Models/PayrollAu/ReimbursementLines.php

```diff
@@ -242,7 +242,7 @@ public function setReimbursementLines($reimbursement_lines)
     #[\ReturnTypeWillChange]
     public function offsetExists($offset)
     {
-        return isset($this->container['reimbursement_lines'][$offset]);
+        return isset($this->container['ReimbursementLines'][$offset]);
```

Comment:
why property has changed to CamelCase?

Reply:
it was a miss defining the key name casing correctly. Now it is fixed to - 'reimbursement_lines'